deps/build.jl

```diff
@@ -75,8 +75,12 @@ mkpath(bin_dir)
 
 function download_and_unpack(url)
     tensorflow_zip_path = joinpath(base, "downloads/tensorflow.tar.gz")
-    download(url, tensorflow_zip_path)
-    run(`tar -xzf $tensorflow_zip_path -C downloads`)
+    try
+        run(`tar -xzf $tensorflow_zip_path -C downloads`)
+    catch
```

Comment:
This `catch`  block is too general. Can you rather explicitly check if the file exists and if so, skip the downloading?